src/traces/histogram/attributes.js

```diff
@@ -82,15 +82,21 @@ module.exports = {
         role: 'style',
         description: [
             'Determines whether or not the x axis bin attributes are picked',
-            'by an algorithm.'
+            'by an algorithm. Note that this should be set to false if you',
```

Comment:
but we should still have some note that expresses you don't _have_ to set this attribute, we'll logically set it for you with the `defaultDepends`